### PR TITLE
PHP 7.2 compatibility

### PR DESCRIPTION
**Pull request by mabar**

### CODE REVIEW

File: src/Kdyby/SessionPanel/Diagnostics/SessionPanel.php [hunk 1]
```diff
@@ -15,9 +15,11 @@
  * @author Pavel Železný <info@pavelzelezny.cz>
  * @author Filip Procházka <email@filip-prochazka.cz>
  */
-class SessionPanel extends Nette\Object implements Tracy\IBarPanel
+class SessionPanel implements Tracy\IBarPanel
 {
 
+	use Nette\SmartObject;
```
Comment (enumag) [inline src/Kdyby/SessionPanel/Diagnostics/SessionPanel.php]: I think we need to raise version constraint for nette as well. This trait does not exist in older versions.

Comment (mabar) [reply src/Kdyby/SessionPanel/Diagnostics/SessionPanel.php]: Fixed. Totally forgot because of missing dependency.

File: composer.json [hunk 2]
```diff
@@ -20,9 +20,10 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.2",
+		"php": ">=5.4.0",
 		"nette/di": "~2.2",
 		"nette/http": "~2.2",
+		"nette/utils": "~2.4",
```
Comment (enumag) [inline composer.json]: Not sure if nette/di 2.2 will work with nette/utils 2.4. Better raise all nette dependencies to 2.4 in my opinion.

Comment (mabar) [reply composer.json]: Ok, did it. Hope this is last commit for that change :D

**Pull request merged**